packages/@ember/-internals/glimmer/index.ts

```diff
@@ -340,6 +340,7 @@
   @method partial
   @for Ember.Templates.helpers
   @param {String} partialName The name of the template to render minus the leading underscore.
+  @deprecated Use `<div hidden={{this.isHidden}}>` or `{{#if this.showComponent}} <MyComponent /> {{/if}}`
```

Comment:
Hmm, this seems like its in the wrong place. This documentation bit is for `partial` I think.

Reply:
Thanks, it seems that my brain can't handle two [concurrent](https://github.com/emberjs/ember.js/pull/18491) PRs.